### PR TITLE
Api/internaltx

### PR DESCRIPTION
**Pull request by dutterbutter**
**Changes made in this pull request:**

- Added internal tx getter functions 
- Added internal tx endpoint to shyftBlockExplorerApi

### CODE REVIEW

File: shyftBlockExplorerApi/handler.go [hunk 1]
```diff
@@ -195,25 +195,37 @@ func GetRecentBlock(w http.ResponseWriter, r *http.Request) {
...
+		http.Error(w, err.Error(), 500)
+		return
+	}
 
 	w.Header().Set("Content-Type", "application/json; charset=UTF-8")
 	w.WriteHeader(http.StatusOK)
 
-	fmt.Fprintln(w, "Get InternalTransactions", address)
+	fmt.Fprintln(w, internalTxs)
 }
```
Comment (GregTheGreek) [inline shyftBlockExplorerApi/handler.go]: Is this function doing anything? It seems like it's not sending the internal tx back

File: shyftBlockExplorerApi/handler.go [hunk 2]
```diff
@@ -195,25 +195,37 @@ func GetRecentBlock(w http.ResponseWriter, r *http.Request) {
...
+	if err != nil {
+		http.Error(w, err.Error(), 500)
+		return
+	}
 
 	w.Header().Set("Content-Type", "application/json; charset=UTF-8")
 	w.WriteHeader(http.StatusOK)
 
-	fmt.Fprintln(w, "Get InternalTransactions", address)
+	fmt.Fprintln(w, internalTxs)
```
Comment (GregTheGreek) [inline shyftBlockExplorerApi/handler.go]: this as well

File: shyftBlockExplorerApi/handler.go (hunk 1 above)
Comment (dutterbutter) [reply shyftBlockExplorerApi/handler.go]: I dont understand? Why dont you think the getter functions and the API handler functions are sending the internals txs back?

Comment (GregTheGreek) [reply shyftBlockExplorerApi/handler.go]: Nevermind i set it up incorrectly.

**Pull request merged**